modules/books.js

```diff
@@ -0,0 +1,63 @@
+/*eslint-disable*/
+import Book from './component.js';
+
+const bookTitle = document.querySelector('#bookName');
+const bookAuthor = document.querySelector('#bookAuthor');
+const addBk = document.querySelector('.add-btn');
+const booksWrapper = document.querySelector('.books-list');
+class BooksdataBase {
+  constructor() {
+    this.booksList = JSON.parse(localStorage.getItem('bookStorage')) || [];
+  }
+
+  addBooks() {
```

Comment:
- [ ] Please make sure that your app is working. At this moment, it is not possible to add and remove a book. The link `Add new` is not working hence I can't add a book. From the requirements of this project, it should be possible to add and remove a book. Please make sure to address this issue.

Reply:
please its working with on my laptop, please kindly check again